src/Elastic.OpenTelemetry/Plugin.cs

```diff
@@ -11,20 +11,20 @@
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
-namespace Elastic.OpenTelemetry.AutoInstrumentationPlugin;
+namespace Elastic.OpenTelemetry;
 
 /// <summary>
 /// Elastic Distribution for OpenTelemetry .NET plugin for Auto Instrumentation.
 /// <para>Ensures all signals are rich enough to report to Elastic</para>
 /// </summary>
 // ReSharper disable once UnusedType.Global
-public class ElasticAutoInstrumentationPlugin
+public class Plugin
```

Comment:
To clarify what this public class does, should we keep the original class name or just `AutoInstrumentationPlugin`?

Reply:
We could I was trying to keep it short for documentation purposes since the environment name needs to FQ type name

```
ENV OTEL_DOTNET_AUTO_PLUGINS="Elastic.OpenTelemetry.Plugin, Elastic.OpenTelemetry, Version=1.0.0.0, Culture=neutral, PublicKeyToken=069ca2728db333c1"
```

I think thats sufficiently self documenting? Wdyt?

Reply:
I'm not overly concerned with the length. It might pop up in intelligence and be confusing, plus I could potentially see us wanting some base plugin class of our own.

Reply:
Cool will amend!